013_IP_L141/013_IP_L141_unsafe.py

```
@@ -0,0 +1,305 @@
+    except (json.JSONDecodeError, FileNotFoundError):
+        pass
+
+    return redirect(url_for('chat_net'))
+
+@app.route('/comment/<int:post_index>', methods=['POST'])
+def comment(post_index):
+    pathlist = os.listdir(POSTS_DIR)
+    file_path = os.path.join(POSTS_DIR, pathlist[post_index])
+    content = request.form.get('comment-content')
+    cookie = request.cookies.get("username")
+    if cookie==None:
```

Ruff (0.14.1)

Comparison to `None` should be `cond is None`

Replace with `cond is None`

(E711)